storagenode/notifications/notifications.go

```diff
@@ -0,0 +1,70 @@
+package notifications
+
+import (
+	"context"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// DB is an implementation of notifications.Notifications.
```

Comment:
it's interface, not implementation
this DB interface exposes the way how to manage records in your db implementation